hera_sim/visibilities/vis_cpu.py

```diff
@@ -298,12 +381,16 @@ def vis_cpu(antpos, freq, eq2tops, crd_eq, I_sky, bm_cube,
     assert ncrd == 3, "crd_eq must have shape (3, NPIX)."
     assert I_sky.ndim == 1 and I_sky.shape[0] == npix, \
         "I_sky must have shape (NPIX,)."
-    bm_pix = bm_cube.shape[-1]
-    assert bm_cube.shape == (
-        nant,
-        bm_pix,
-        bm_pix,
-    ), "bm_cube must have shape (NANTS, BM_PIX, BM_PIX)."
+    
```

Comment:
Should check that either `beam_list` or `bm_cube` is not None and give a useful error.